frontend/site-registry/src/app/features/details/disclosure/Disclosure.tsx

```diff
@@ -271,9 +283,10 @@ const Disclosure = () => {
                         }
                         </div>
                     </Widget>
+                    }
                 </div>
             </div>
-            { userType === UserType.Internal && <p className="sr-time-stamp">{formData.srTimeStamp}</p>}
+            { userType === UserType.Internal && <p className="sr-time-stamp">{formData.srTimeStamp ?? 'Hard Code Value'}</p>}
```

Comment:
why do we have this hard code value ?

Reply:
Keeping it for future reference because right now sr time stamp is not confirmed